controllers/btpoperator_controller.go

```diff
@@ -1480,6 +1480,7 @@ func (r *BtpOperatorReconciler) prepareCertificatesReconciliationData(ctx contex
 		return err
 	}
 	if certificatesRegenerationDone {
+		r.metrics.IncreaseCertsRegenerationsCounter()
```

Comment:
Why don't we instrument the `doFullCertificatesRegeneration` with metrics.

Reply:
There is also a `doPartialCertificatesRegeneration` that is independent of `doFullCertificatesRegeneration`

Reply:
I could move this to `ensureCertificatesExists` before the return statements, where we signal that a regen has occurred

Reply:
Leave it as it is please.